docs/LocalMapDTO.md

```diff
@@ -0,0 +1,79 @@
+###LocalMapDTO
+Example:
+ ```json
+{
+	"id":"exampleId",
+	"name":"exampleName",
+	"coordinates":{
+		"lat":12.22,
+		"lon":13.33
+	}
+}
```

Comment:
do we put somewhere information about the distance between localization of local and client localization? we can display it somewhere

Reply:
We should think about whether they should be returned by the same dto